### PR TITLE
fix: Payment Terms changes not applied in the B2B Checkout Settings

### PR DESCRIPTION
**Pull request by icaroov**
#### What problem is this solving?
Updating payment terms in the organization details UI does not apply to B2B checkout settings.

This behavior occurs because the update occurs only for the organization, but B2B checkout settings use cost center information, which is not updated by the organization details UI.

#### How to test it?
* Create an organization;
* In the organization details UI, update the payment terms;
* Access the website, assemble a cart and access checkout; Payment terms will remain the same.

[Workspace](https://icaroov--b2bstore005.myvtex.com/checkout/#/cart)

### CODE REVIEW

**Comment by icaroov** [comment]
This current PR replaces and fixes the tests of this other PR: https://github.com/vtex-apps/b2b-checkout-settings/pull/68.

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+- Adjust payment terms on cost center
+
```
Comment (enzomerca) [inline CHANGELOG.md:19]: We need to move these lines to under the `##[Unreleased]` section (before `## [1.11.2] - 2024-06-12`

Comment (icaroov) [reply CHANGELOG.md:23]: Hey @enzomerca, changes made!!

**Review by enzomerca — APPROVED** [review]

**Comment by enzomerca** [comment]
Approved. Will hold the deployment until next week.

**Pull request merged**